src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,29 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  width: $paragraph-width;
```

Comment:
The variable `$paragraph-width` is used here. Ensure that this variable is defined in the `variables` file or elsewhere in the project.